Eshop.Domain/Category.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Eshop.Domain
+{
+    public class Category
+    {
+        public Category(string categoryName, int id, string description)
+        {
+            this.id = id;
+            this.description = description;
+        }
+        public string categoryName { get;  set; }
+        public int id { get; private set; }
+        public string description { get; set; }
+
+        public void Update(string categoryName)
+        {
+            if (string.IsNullOrEmpty(categoryName))
+            {
+                throw new ArgumentNullException("Can not update an empty name");
+            }
+
+            categoryName = categoryName;
```

Comment:
Here you did not test it your code - because this is bug even VS is helping you.
Try to guess why. Very important
![image](https://github.com/akonaweb/Eshop/assets/162015065/f3ba2ae1-900c-49f3-99fe-c10f77179e56)

Reply:
fixed, not even VS is trying to show me